cocotb/share/makefiles/Makefile.deprecations

```diff
@@ -5,3 +5,7 @@
 ifdef COCOTB_NVC_TRACE
    $(warning COCOTB_NVC_TRACE is deprecated, see the "Simulator Support" section in the documentation.)
 endif
+
+ifdef VERILATOR_TRACE
+   $(warning VERILATOR_TRACE is deprecated, see the "Simulator Support" section in the documentation.)
```

Comment:
Do we also use three-space indents in other makefiles? Seems unusual.

Reply:
No, I think this one is unique in that regard.

Reply:
Oops, fixed.